README.md

```diff
@@ -36,6 +38,17 @@ pip install -r syndecrypt/requirements.txt
 python Synology.py
 ```
 
+## Installing Compile-time Dependencies
+
+If you choose to install dependencies manually per the instructions above, you will need to build some dependencies from source. If `pip install` is dying with clang errors during this process, try following these steps:
```

Comment:
Could you change this to `If you choose to install dependencies manually as per the instructions above, you may need to build some dependencies from source.`